docker-compose-dev.yml

```diff
@@ -16,13 +16,10 @@ services:
       context: ../test-ehr
       dockerfile: Dockerfile.keycloak
 
-  # Create test ehr container
```

Comment:
Why were all of the comments removed?

Reply:
which comments?

Reply:
The diff shows comments that were there like:
"# Create test her container"
"# Name of our service"

Reply:
okay, I'll add those comments back in